examples/basics/custom_embeddings_sdk.ipynb

```diff
@@ -39,7 +39,7 @@
     {
```

Comment:
<p>Can you seperate out the dataset id in a different cell like in this notebook this: <a href="https://github.com/Labelbox/labelbox-python/blob/develop/examples/basics/datasets.ipynb" target="_blank">https://github.com/Labelbox/labelbox-python/blob/develop/examples/basics/datasets.ipynb</a></p><p> </p><p>Also side note while we are at it in that notebook I linked can you remove the dataset id</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1582/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Done